DEVELOPMENT.md

```diff
@@ -0,0 +1,51 @@
+# React Router Development
+
+## Releases
+
+New releases should be created from release branches originating from the `dev` branch. When you are ready to begin the release process:
+
+- Check out the `dev` branch
+- Make sure you have all of the changes from GitHub
+- Create a new release branch with the `release-` prefix (eg, `git checkout -b release-v6.5.1`)
+  - **IMPORTANT:** The `release-` prefix is important, as this is what triggers our GitHub CI workflow that will ultimately publish the release.
+
+Changesets will do most of the heavy lifting for our releases. When changes are made to the codebase, an accompanying changeset file should be included to document the change. Those files will dictate how Changesets will version our packages and what shows up in the changelogs.
+
+### Starting a new pre-release
+
+- Ensure you are on the new `release-*` branch
+- Enter Changesets pre-release mode using the `pre` tag: `yarn changeset pre enter pre`
```

Comment:
Is the final argument to `yarn changeset pre enter {tag}` the verbatim tag you'll get on npm?  If so - should we update to: 

```suggestion
- Enter Changesets pre-release mode using the `pre` tag: `yarn changeset pre enter X.Y.Z-pre.N`
```

Reply:
No, because the actual tag depends on the version number and packages may be versioned independently. Changesets will iterate and tag for us as expected.

Reply:
Pre-releases are a bit weird, but I tested it one of my own repos and it basically works like this (assume here that packages are not version-fixed):

```sh
# current versions
package-a@1.2.0
package-a@1.4.5
```

```sh
yarn changeset pre enter pre
```

After we version, assume `package-a` is a minor bump and `package-b` is a patch, we'll get:

```sh
# current versions
package-a@1.3.0-pre.0
package-a@1.4.6-pre.0
```